### ISSUE TITLE
vcffilter -v option is not working

### ISSUE THREAD

**Issue by SusanaFreitas** [vcflib/vcflib#155]
Hey!
I have been recently using vcffilter with the -v option
vcffilter -v -f "TYPE = ins" vcf.vcf > out.vcf
and this seems to make exactly the same output as:
vcffilter -f "TYPE = ins" vcf.vcf > out.vcf

Am I doing something wrong or is it a problem with this option in particular?
Cheers

**Comment by zeeev** [comment]
This works. 

 ~/tools/vcflib/bin/vcffilter  -f "! ( SVTYPE = INS )"  HG005.whamg.vcf

**Comment by SusanaFreitas** [comment]
of course... thanks, I forgot about that.
Cheers
Susana

2016-06-06 20:58 GMT+01:00 Zev Kronenberg notifications@github.com:

**Comment by zeeev** [comment]
It's not intuitive.

**Comment by SusanaFreitas** [comment]
Hello!

sorry for repeating the question, but I still can't make it...
I have been trying:
vcffilter -f "! ( SVTYPE = INS )"  var22_71.vcf > var22_71_ins.vcf
and what I get is basically the same file as the input. I also tried
vcffilter -f "! ( TYPE = INS )"  var22_71.vcf > var22_71_ins.vcf
vcffilter -f "! ( TYPE = ins )"  var22_71.vcf > var22_71_ins.vcf

because I did not know whether this would matter, and it didn't. What I am
doing now is selecting all positions that I don't want to keep (ins,del and
complex) and remove them with vcftools, but it takes more time (and more
prone to errors). Is it my problem? Am I doing something wrong?
Just to make sure it wasn't a problem with my vcflib version, I
re-installed it and the error persists.

Thanks!!!
Susana

2016-06-06 21:52 GMT+01:00 Zev Kronenberg notifications@github.com:
